src/corelibs/U2Core/src/models/DocumentModel.h

```diff
@@ -340,9 +340,10 @@ enum DocumentObjectRemovalMode {
     DocumentObjectRemovalMode_Deallocate,  // ordinary removal: both object and its DB representation are deallocated
     DocumentObjectRemovalMode_OnlyNotify,  // fake removal: neither object nor its DB representation are deallocated.
     // Only signals about removal are emitted. The object itself remains to belong to the document.
-    DocumentObjectRemovalMode_Release  // fake removal: the same as DocumentObjectRemovalMode_OnlyNotify.
+    DocumentObjectRemovalMode_Release,  // fake removal: the same as DocumentObjectRemovalMode_OnlyNotify.
     // Additionally, the object is excluded from the document's child objects list.
     // External code has to handle a proper object deallocation.
+    DocumentObjectRemovalMode_Silent  // sudo removes an object from the doc, removes an object, leaving it in the db.
```

Comment:
The better name would be a 'Detach'. 
What does 'sudo' in the comment mean?

Reply:
sudo as in a Unix shell. It means "delete no matter what".